gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/Transport.java

```diff
@@ -20,12 +20,13 @@ public enum Transport {
   GRPC_REST;
 
   /**
-   * Parse command line transport argument in the format `grpc+rest`.
+   * Parse command line transport argument in the format `grpc+rest`. Convert name String to
+   * lowercase as `gRPC` is not valid but `grpc` is
```

Comment:
This comment is a little confusing as `gRPC` is a valid input now, we might want to just remove this comment and update the valid inputs section in the javadoc below.

Reply:
Ah I see. I added that comment to explain why I was calling `toLowercase()`, but I see why that's confusing now. I'll change.